Makefile

```diff
@@ -19,9 +19,7 @@ test-unit: clean
 
 .PHONY: package
 package:
-ifndef version
-	$(error No version given. Aborting)
-endif
+	version="unversioned"
 	$(info Packaging version: $(version))
```

Comment:
When this is deployed to `platform` you would want a check here for the `version` variable rather than setting it to `unversioned` as that is only useful for local development. Can you revert to the original here?